docs/cli.md

```diff
@@ -503,3 +503,24 @@ gcli tag update --tag tagA --rename newTag
 ```bash
 gcli tag update --tag tagA --comment "new comment"
 ```
+
+### Owners commands
+
+#### List an owner
+
+```bash
+gcli catalog details --name postgres --owner
+```
+
+#### Set an owner to a user
+
+```bash
+gcli catalog update --name postgres --user admin --owner
```

Comment:
This seems odd; can we just use "set" command? considering already use "set" for tag, property, owner is just another type. e.g, "gcli catalog set --name postgres --owner admin --owner-type user".

Another possible case is using different commands like "set-tag", "set-owner", with an extensible properties. for example:
"gcli catalog set-owner --name postgres --properties owner=admin;type=user".

Please think about it.

Reply:
This was chosen on purpose. If we used the `set` action, that would result in duplicating a lot of code as each handle command method would need to deal with it.

Reply:
Further complications with using the `set` command approach:
- The user would expect `remove` to work, it would not be supported as there is no API to remove an owner
- We may need to add two options rather than `--owner-user` and `--owner-group`

Reply:
It needs to be merged as it is block other PRs

Reply:
Change to use set command - you'll see why I was reluctant to do it this way

Reply:
There could be less code duplication if we required "--owner" to the set commands.